### PR TITLE
S02us01 03 button to go up

### LINKED ISSUE #21: US01-03: Création d'un bouton pouvant être appelé sur les différentes pages

### PR DESCRIPTION
**Pull request by bouboumael**
Create a banner to include it in all pages and be able to modify the title and image

### CODE REVIEW

**Review by ChristianOlmedo — DISMISSED** [review]

File: src/View/Includes/_btnToGoUp.html.twig [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<a href="" class="btn-up"><svg xmlns="http://www.w3.org/2000/svg" width="52" height="50" fill="var(--light)" class="bi bi-arrow-up rounded p-2 position-fixed bottom-0 end-0 me-3 mb-3 d-none d-sm-none d-md-block" viewBox="0 0 16 16" style="background-color:var(--primary-color)">
+  <path fill-rule="evenodd" d="M8 15a.5.5 0 0 0 .5-.5V2.707l3.146 3.147a.5.5 0 0 0 .708-.708l-4-4a.5.5 0 0 0-.708 0l-4 4a.5.5 0 1 0 .708.708L7.5 2.707V14.5a.5.5 0 0 0 .5.5z"/>
+</svg></a>
```
Comment (LucyMerlier) [inline src/View/Includes/_btnToGoUp.html.twig]: "_btnToGoUp" => not a valid file name (abbreviation + should be snake_case)

File: src/View/layout.html.twig [hunk 2]
```diff
@@ -1,25 +1,28 @@
...
 
-    <title>{% block title %}Haras du Val de Loire{% endblock %}</title>
-    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-eOJMYsd53ii+scO/bJGFsiCZc+5NDVN2yr8+0RDqr0Ql0h+rP48ckxlpbzKgwra6" crossorigin="anonymous">
-    <link rel="stylesheet" href="/assets/css/style.css">
-    {% block stylesheet %}{% endblock %}
-    <link rel="icon" href="/assets/images/logo_favicon.png">
+	<title>
+		{% block title %}Haras du Val de Loire
+		{% endblock %}
+	</title>
```
Comment (LucyMerlier) [inline src/View/layout.html.twig]: why write it that way?

File: src/View/layout.html.twig [hunk 3]
```diff
@@ -1,25 +1,28 @@
...
-        {% include 'Includes/_navbar.html.twig' %}
-    {% endblock %}
-    {% block content %}{% endblock %}
-    {% block javascript %}{% endblock %}
-    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta3/dist/js/bootstrap.bundle.min.js" integrity="sha384-JEW9xMcG8R+pH31jmWH6WWP0WintQrMb4s7ZOdauHnUtxwoG2vI5DkLtS3qm9Ekf" crossorigin="anonymous"></script>
+	{% block navbar %}
+		{% include 'Includes/_navbar.html.twig' %}
+	{% endblock %}
+	{% block content %}{% endblock %}
+	{% block btnUp %}
```
Comment (LucyMerlier) [inline src/View/layout.html.twig]: "btnUp" => not a valid name

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comments

File: src/View/Includes/_btnToGoUp.html.twig (hunk 1 above)
Comment (bouboumael) [reply src/View/Includes/_btnToGoUp.html.twig]: ok correction done

File: src/View/layout.html.twig (hunk 2 above)
Comment (bouboumael) [reply src/View/layout.html.twig]: Done but the title should stay if that was the question?

File: src/View/layout.html.twig (hunk 3 above)
Comment (bouboumael) [reply src/View/layout.html.twig]: change name ok

**Review by ChristianOlmedo — APPROVED** [review]

**Review by LucyMerlier — APPROVED** [review]

**Pull request merged**